res/values-pt-rBR/strings.xml

```diff
@@ -153,7 +153,7 @@
     <string name="Dot_Gobbler">"Guloso de Pontos"</string>
     <string name="Dot_Devourer">"Devorador de Pontos"</string>
     <string name="Dot_Gormandizer">"Gordice de Pontos"</string>
-    <string name="Dot_Sommelier">"Escançador de Pontos"</string>
+    <string name="Dot_Sommelier">"Apreciador de Pontos"</string>
```

Comment:
"Escançador" was already correct (translation for "sommelier"), please return it.